src/main/java/ratpack/zipkin/ResponseTagCustomizer.java

```diff
@@ -0,0 +1,18 @@
+package ratpack.zipkin;
+
+import ratpack.func.Pair;
+import ratpack.http.Response;
+import ratpack.path.PathBinding;
+
+import java.util.List;
+
+@FunctionalInterface
+public interface ResponseTagCustomizer {
+  /**
+   * Return the tags for a response and pathbinding.
+   * @param response the response
+   * @param pathBinding the path binding
+   * @return a list of key/value pairs to add to the span as tags.
+   */
+  List<Pair<String, String>> tags(Response response, PathBinding pathBinding);
```

Comment:
here's a couple things to consider, which may not be important enough, but whatev :)

from a microperf pov, accepting a SpanCustomizer or otherwise is better than returning a list except in the case of empty list. even in empty list, it is no difference unless you create an iterator. for example, list is nice as you can use  `for (int i.. ` syntax to scan the list which doesn't allocate an iterator and benchmarks faster.

The design of passing a `SpanCustomizer` as a final arg allows you to skip iteration completely. Ex you just push things in that you want.

Ex. below will bench much faster..
```
if (foo) customizer.tag("foo", foo);
```
than this
```
result = new ArrayList();
if (foo) result.add(entry("foo", foo));
return result;
}

... somewhere else..
tags = parser.tags(...)
for (entry tag: tags) span.tag(entry._1, entry_.2)
```